src/fides/api/ctl/routes/health.py

```diff
@@ -74,12 +76,15 @@ async def health(
     """Confirm that the API is running and healthy."""
     database_health = get_db_health(CONFIG.database.sync_database_uri, db=db)
     cache_health = get_cache_health()
-    response = {
+    response: Dict[str, Any] = {
         "webserver": "healthy",
         "version": str(fides.__version__),
         "database": database_health,
         "cache": cache_health,
     }
+    fides_is_using_workers = not celery_app.conf["task_always_eager"]
```

Comment:
@seanpreston what do you think about updating this so that it returns `"workers": "disabled"` or something if no workers present? just a thought